build-system/tsconfig.json

```diff
@@ -27,15 +27,50 @@
     "./common/**/*.ts",
     "./server/**/*.js",
     "./server/**/*.ts",
-    "./tasks/**/*.ts",
-    "./tasks/**/*.ts"
+    "./tasks/*.js",
+    "./tasks/*.ts",
+    "./tasks/bundle-size/*.js",
+    "./tasks/bundle-size/*.ts",
+    "./tasks/coverage-map/*.js",
+    "./tasks/coverage-map/*.ts",
+    "./tasks/create-golden-css/*.js",
+    "./tasks/create-golden-css/*.ts",
+    "./tasks/css/*.js",
+    "./tasks/css/*.ts",
+    // "./tasks/e2e/*.js", // 253 remaining type errors.
+    // "./tasks/e2e/*.ts",
+    "./tasks/extension-generator/*.js",
+    "./tasks/extension-generator/*.ts",
+    "./tasks/get-zindex/*.js",
+    "./tasks/get-zindex/*.ts",
+    "./tasks/markdown-toc/*.js",
+    "./tasks/markdown-toc/*.ts",
+    // "./tasks/performance/*.js", // 35 remaining type errors.
+    // "./tasks/performance/*.ts",
+    "./tasks/prepend-global/*.js",
+    "./tasks/prepend-global/*.ts",
+    "./tasks/release/*.js",
+    "./tasks/release/*.ts",
+    "./tasks/runtime-test/*.js",
+    "./tasks/runtime-test/*.ts",
+    "./tasks/storybook/*.js",
+    "./tasks/storybook/*.ts",
+    "./tasks/sweep-experiments/*.js",
+    "./tasks/sweep-experiments/*.ts",
+    // "./tasks/visual-diff/*.js", // 12 errors due to async imports.
+    // "./tasks/visual-diff/*.ts",
+    "./test-configs/**/*.js",
+    "./test-configs/**/*.ts",
```

Comment:
This change is a more accurate representation of the state of the type checking effort but eventually it will be be reverted to the more general includes list on previously here. I am fine with not updating this at all if others dislike the churn.